src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+}
+
```

Comment:
remove redundant empty lines

Reply:
I don't see any empty lines, can you show me more precisely?